packages/smooth_app/lib/themes/smooth_theme.dart

```diff
@@ -56,11 +56,18 @@ class SmoothThemes {
       brightness: Brightness.light,
     );
 
-    ColorScheme myColorScheme = isDarkTheme ? colorDark : colorLight;
+    final ColorScheme myColorScheme = isDarkTheme ? colorDark : colorLight;
 
     return ThemeData(
       colorScheme: myColorScheme,
-      buttonColor: isDarkTheme ? const Color(0xFF696464) : Colors.black,
+      buttonTheme: ButtonThemeData(
+        textTheme: ButtonTextTheme.accent,
+        colorScheme: Theme.of(context).colorScheme.copyWith(
+              primary: isDarkTheme ? const Color(0xFF696464) : Colors.black,
```

Comment:
I think you should be able to set the primary directly in the `ColorScheme`, without having to use `copyWith`.